commands/support_bundle.go

```diff
@@ -11,19 +10,23 @@ import (
 	"github.com/jfrog/jfrog-support-bundle-flunky/commands/actions"
 	"github.com/jfrog/jfrog-support-bundle-flunky/commands/http"
 	"os"
-	"strconv"
 	"strings"
 	"time"
 )
 
 const (
-	serverIDFlag        = "server-id"
-	targetServerIDFlag  = "target-server-id"
-	downloadTimeoutFlag = "download-timeout"
-	retryIntervalFlag   = "retry-interval"
-	promptOptionsFlag   = "prompt-options"
-	cleanupFlag         = "cleanup"
-	targetRepoFlag      = "target-repo"
+	serverIDFlag       = "server-id"
```

Comment:
Are you sure we want to make the constants public?

Reply:
That's the only way I found to test most of the command code in integration tests.

Reply:
hard code the values in the test.

Reply:
Right, I'll do that